src/components/dynamic-lists/beagle-list-view/beagle-list-view.component.html

```diff
@@ -0,0 +1,32 @@
+<!-- 
+  * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *  http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License. 
+-->
+
+<beagle-dynamic-list 
+  type='LIST'
+  [direction] = direction
+  [dataSource] = dataSource
+  [iteratorName] = iteratorName
+  [template] = template
+  [onInit] = onInit
+  [onScrollEnd] = onScrollEnd
+  [scrollEndThreshold] = scrollEndThreshold
+  [useParentScroll] = useParentScroll
+  [key] = key
+  [__suffix__]= __suffix__
+  [isScrollIndicatorVisible] = isScrollIndicatorVisible
+  [parentReference] = this>
```

Comment:
Please follow the HTML patterns, even if it is Angular.

```suggestion
<beagle-dynamic-list 
  type="listType.List"
  [direction]="direction"
  [dataSource]="dataSource"
  [iteratorName]="iteratorName"
  [template]="template" 
  [onInit]="onInit"
  [onScrollEnd]="onScrollEnd"
  [scrollEndThreshold]="scrollEndThreshold"
  [useParentScroll]="useParentScroll"
  [key]="key"
  [__suffix__]="__suffix__"
  [isScrollIndicatorVisible]"="isScrollIndicatorVisible"
  [parentReference]="this">
```